linkerd/app/core/src/svc.rs

```diff
@@ -202,6 +195,17 @@ impl<S> Stack<S> {
         self.push(http::insert::NewResponseInsert::layer())
     }
 
+    pub fn push_buffer_on_service<Req>(
```

Comment:
tiny nit, unimportant: `push_buffer` has a doc comment, should we put one on this too? it's pretty obvious what it does from the name if the reader understands what `push_buffer` does, so idk if it's important...